demos/interactive_face_detection_demo/visualizer.hpp

```diff
@@ -96,9 +96,9 @@ class Visualizer {
         size_t frameIdx;
     };
 
-    explicit Visualizer(cv::Size const& imgSize, int leftPadding = 10, int rightPadding = 10, int topPadding = 75, int bottomPadding = 10);
+    explicit Visualizer(int leftPadding = 10, int rightPadding = 10, int topPadding = 75, int bottomPadding = 10);
 
-    void enableEmotionBar(std::vector<std::string> const& emotionNames);
+    void enableEmotionBar(const cv::Size inImgSize, std::vector<std::string> const& emotionNames);
```

Comment:
What was the reason to move the inImgSize here?

Reply:
To make emotion bar enabled we need to now frame size.
In OMZ  version frame size is one of visualizer constructor parameters
And frame size can be found out before starting video processing

In G-API pipeline we can find out the frame size only after starting streaming.
So we have to enable emotion bar only after starting video processing (after starting streaming, after obtaining first frame)
Because of it  frame size should be one of the `enableEmotionBar()` parameters

Reply:
I will follow-up on this in the #6.